### DISCUSSION TITLE
[CS2] Attempting to extract and modify 'botprofile.db' - CS2 still uses original from VPK archive

### DISCUSSION THREAD

**Discussion by GTManiK** [SteamDatabase/ValveResourceFormat#642] [General]
Guys, is there any way to make CS2 load an extracted 'botprofile.db'? When I extract it, modify it and then place inside .\game\csgo folder (directory structure corresponds to file location in VPK archive), it seems that CS2 ignores this file and loads the original one from VPK. Is there a way to override content in VPKs by using extracted files from them? How does it work?

Many thanks!

**Comment by kristiker** [comment]
Make a map addon
